client/src/component/ReviewsEntry.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+class ReviewsEntry extends React.Component {
```

Comment:
I think the naming could probably be improved here. Maybe call the parent Reviews and the children Review? Or ReviewsList for the parent if you want to make it more obvious. Having a plural in the name of the children confused me a bit.